lisa/sut_orchestrator/azure/features.py

```diff
@@ -392,6 +397,10 @@ def _generate_min_capability(self, capability: Any) -> Any:
         # all caching types are supported, so just take the value from requirement.
         min_value.data_disk_caching_type = self.data_disk_caching_type
 
+        min_value.has_resource_disk = search_space.generate_min_capability_boolean(
```

Comment:
It's hard to define what's the "min" for a boolean. So it doesn't like shareable logic.